fuzzit.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+set -xe
+
+# first argument must be:
+# fuzzing          : upload to fuzzit.dev servers for continuous fuzzing
+# local-regression : run a short regression locally using past crashing test cases
+
+# go-fuzz doesn't support modules yet, so ensure we do everything
+# in the old style GOPATH way
+export GO111MODULE="off"
+
+# install go-fuzz
+go get -u github.com/dvyukov/go-fuzz/go-fuzz github.com/dvyukov/go-fuzz/go-fuzz-build
+
+# target name can only contain lower-case letters (a-z), digits (0-9) and a dash (-)
+# to add another target, make sure to create it with `fuzzit create target`
+# before using `fuzzit create job`
+TARGETS=("promql-parse-metric" "promql-parse-open-metric" "promql-parse-metric-selector" "promql-parse-expr")
+
+go-fuzz-build -libfuzzer -func FuzzParseMetric -o promql-parse-metric.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseOpenMetric -o promql-parse-open-metric.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseMetricSelector -o promql-parse-metric-selector.a ./promql
+go-fuzz-build -libfuzzer -func FuzzParseExpr -o promql-parse-expr.a ./promql
+
+for TARGET in "${TARGETS[@]}"
+do
+    clang -fsanitize=fuzzer ${TARGET}.a -o ${TARGET}
+done
+
+# install fuzzit for talking to fuzzit.dev service
+# or latest version:
+# https://github.com/fuzzitdev/fuzzit/releases/latest/download/fuzzit_Linux_x86_64
+wget -q -O fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.23/fuzzit_Linux_x86_64
```

Comment:
Many places don't install `wget` by default. `curl -s -o fuzzit https://github.com/fuzzitdev/fuzzit/releases/download/v2.4.23/fuzzit_Linux_x86_64` usually brings better luck.

Reply:
I can change this.

Reply:
done.

Reply:
The curl command doesnt work:( so I changed it back to `wget` it's available in travis so I hope it's not a problem.